inc/Widget/SmailyWidget.php

```diff
@@ -119,20 +118,21 @@ public function widget( $args, $instance ) {
 				<input type="hidden" name="failure_url" value="' . esc_url( $return_url ) . '" />
 				<input type="hidden" name="language" value="' . esc_html( $lang ) . '" />
 		';
+
 		// Optional autoresponder when selected.
 		echo $autoresponder_id ? '<input type="hidden" name="autoresponder" value="' . esc_html( $autoresponder_id ) . '" />' : '';
 		echo '</div>
 				<p>
-					<label>' . esc_html__( 'Email', 'smaily' ) .'</label>
+					<label>' . esc_html_e( 'Email', 'smaily' ) .'</label>
 					<input type="text" name="email" value="" />
 				</p>
 				<p>
-					<label>' . esc_html__( 'Name', 'smaily' ) . '</label>
+					<label>' . esc_html_e( 'Name', 'smaily' ) . '</label>
 					<input type="text" name="name" value="" />
 				</p>
 				<p>
 					<button class="ui pink basic button" type="submit">' .
-						esc_html__( 'Subscribe', 'smaily' ) .
+						esc_html_e( 'Subscribe', 'smaily' ) .
```

Comment:
Usage of `esc_html__` was correct here, because `esc_html_e` will echo the string when string is actually concatenated.

Consider this example:
```php
$foo = function() { echo 'abc'; };
echo 'xyz' . $foo();
```

You'd expect the output to be `xyzabc`, but the output is actually `abcxyz`, because `$foo` function is executed before `echo 'xyz'` and it echos `abc` first.

You can see the same effect when you look at the widget on the site:
![Screenshot from 2021-05-25 08-41-03](https://user-images.githubusercontent.com/6755852/119445438-8196eb80-bd35-11eb-81da-24ddee6b1b0b.png)